packages/alfa-rules/src/sia-r21/rule.ts

```diff
@@ -40,7 +40,7 @@ export default Rule.Atomic.of<Page, Attribute>({
 
         return {
           1: expectation(
-            hasExplicitRole()(owner),
+            hasRole()(owner),
```

Comment:
Sanshikan is allowing defaulting to implicit role, copying it here.
https://github.com/Siteimprove/sanshikan/blob/master/rules/sia-r21.md#expectation
and second passed example.